packages/x-date-pickers/src/internals/components/PickersPopper.tsx

```diff
@@ -334,13 +309,16 @@ export const PickersPopper = (props: PickerPopperProps) => {
               {...otherPaperProps}
             >
               {children}
-              <PickersPopperAction ownerState={ownerState}>
-                {clearable && (
-                  <Button data-mui-test="clear-action-button" onClick={onClear}>
-                    {clearText}
-                  </Button>
-                )}
-              </PickersPopperAction>
+              {components?.ActionBar && (
+                <components.ActionBar
+                  onAccept={onAccept}
+                  onClear={onClear}
+                  onCancel={onCancel}
+                  onSetToday={onSetToday}
+                  actions={['clear']}
```

Comment:
We will have to be careful here
If the user sets `props.componentsProps.actionBar.onCancel` for instance, I would expect it to call both our callback passed from `onPickerState` AND it's custom callback (like we do on events like `onClick` on the grid).

We could create a small wrapper handling that
Or at least remove those 4 `onXXX` prop from the typing to avoid the temptation of overriding them.

Reply:
I would expect the same, but for now, I simply remove them from the type with `Omit<PickersActionBarProps, 'onAccept' | 'onClear' | 'onCancel' | 'onSetToday'>;` such that it is clearly visible which prop are not handled yet

Let see first how this slot will be used

Reply:
With https://github.com/mui/mui-x/pull/4758 you can do `Omit<PickerActionBarProps, keyof PickerStateProps>`